app/models/job.rb

```diff
@@ -13,4 +13,12 @@ def assign_tech
     tech_matches = Technology.where(name: raw_technologies)
     self.technologies = tech_matches
   end
+
+  def company_name
```

Comment:
These methods are being used for view logic - we might want to consider using [decorators](http://johnotander.com/rails/2014/03/07/decorators-on-rails/) or [View Objects](http://www.jetthoughts.com/blog/tech/2014/08/14/cleaning-up-your-rails-views-with-view-objects.html)